src/Tizen.NUI/src/public/Visuals/VisualObject/GradientVisual.cs

```diff
@@ -0,0 +1,227 @@
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 StartPosition
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.StartPosition, value);
+            }
+            get
+            {
+                Vector2 ret = new Vector2();
+                using (var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.StartPosition))
+                {
+                    propertyValue?.Get(ret);
+                }
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 EndPosition
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.EndPosition, value);
+            }
+            get
+            {
+                Vector2 ret = new Vector2();
+                using (var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.EndPosition))
+                {
+                    propertyValue?.Get(ret);
+                }
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 Center
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.Center, value);
+            }
+            get
+            {
+                Vector2 ret = new Vector2();
+                using (var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.Center))
+                {
+                    propertyValue?.Get(ret);
+                }
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float Radius
+        {
+            set
+            {
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.Radius, value);
+            }
+            get
+            {
+                float ret = 0.0f;
+                using var propertyValue = GetCachedVisualProperty((int)Tizen.NUI.GradientVisualProperty.Radius);
+                propertyValue?.Get(out ret);
+                return ret;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<float> StopOffset
+        {
+            set
+            {
+                stopOffset = value;
+                PropertyArray stopOffsetArray = new PropertyArray();
+                foreach (float val in stopOffset)
+                {
+                    stopOffsetArray.Add(new PropertyValue(val));
+                }
+
+                UpdateVisualProperty((int)Tizen.NUI.GradientVisualProperty.StopOffset, stopOffsetArray);
+            }
+            get
+            {
+                return stopOffset;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<Vector4> StopColor
```

Comment:
Recently NUI avoid to using class type of vectors. For the `Vector4`, we do provide `UIColor` to extress color.